reference/performances_guide/index.rst

```diff
@@ -11,9 +11,25 @@ The amount of data the PIM handles can evolve for each new minor version, thank
 
     There is an early version of this chapter, we'll continue to complete it with more use cases.
 
+.. _changelog: https://github.com/akeneo/pim-community-dev/blob/1.4/CHANGELOG-1.4.md
+
+.. note::
+
+    *Akeneo PIM* has grown and evolved quickly. We added a lot of useful and cool features since the 1.0 release. At this time, *Akeneo PIM* was mostly used
+    by companies with a small and simple catalog. That completely changed. Now our customers and users have more and more products, attributes, families or categories
```

Comment:
It has completely changed. 
[...] have more and more products, attribute, families AND categories.